src/test/ui/NavbarTestPage.ts

```diff
@@ -0,0 +1,40 @@
+import { ChromeDriver } from './ChromeDriver';
+import { WebDriver } from 'selenium-webdriver';
+
+export class NavbarTestPage extends ChromeDriver {
+
+    constructor(driver: WebDriver) {
+        super(driver);
+    }
+
+    async getLoginButtonText(): Promise<string> {
+        const linkText = 'Login'; 
+        return await this.getTextByLinkText(linkText);
+    }
+
+    async getLogoutButtonText(): Promise<string> {
+        const linkText = 'Logout'; 
+        return await this.getTextByLinkText(linkText);
+    }
+
+    async clickHomeButton(): Promise<void> {
+        const element = 'Home';
+        await this.clickByLinkText(element);
+    }
+    async clickJobsButton(): Promise<void> {
+        const element = 'Jobs';
+        await this.clickByLinkText(element);
+    }
+    async clickAboutUsButton(): Promise<void> {
+        const element = 'About Us';
+        await this.clickByLinkText(element);
```

Comment:
About us page doesn't exist and button has been removed from the navbar so no need to test it